version_parser.py

```diff
@@ -0,0 +1,42 @@
+import os
+import subprocess
+
+
+VERSION_PY_PATH = 'gbp/version.py'
+
+
+def _parse_changelog():
+    """Get version from debian changelog and write it to gbp/version.py"""
+    popen = subprocess.Popen('dpkg-parsechangelog', stdout=subprocess.PIPE)
+    out, ret = popen.communicate()
+    for line in out.decode('utf-8').split('\n'):
+        if line.startswith('Version:'):
+            version = line.split(' ')[1].strip()
+            return version
+
+    raise ValueError('Could not parse version from debian/changelog')
+
+
+def _save_version_py(version):
+    with open(VERSION_PY_PATH, 'w') as f:
+        f.write('"The current gbp version number"\n')
+        f.write('gbp_version = "%s"\n' % version)
+
+
+def _load_version():
+    with open(VERSION_PY_PATH, 'r') as f:
+        version_py = f.read()
+    version_py_globals = {}
+    exec(version_py, version_py_globals)
```

Comment:
No exec here please.

Reply:
Why? It's the simplest thing we can do here and it will validate against `SyntaxError` or other problems, so the module can be imported. Any other parsing (regex or hunting for quotes) would be more complex.